src/library_gl.js

```diff
@@ -1400,6 +1403,180 @@ var LibraryGL = {
     GLctx.bufferSubData(target, offset, HEAPU8.subarray(data, data+size));
   },
 
+  // Queries EXT
+  glGenQueriesEXT__sig: 'vii',
+  glGenQueriesEXT: function(n, ids) {
+    for (var i = 0; i < n; i++) {
+      var query = context.disjointTimerQueryExt.createQueryEXT();
```

Comment:
To be compatible with the closure compiler minification, this should be `context.disjointTimerQueryExt['createQueryEXT']();` instead. Likewise for all cases below where a member function of `context.disjointTimerQueryExt` is accessed. Leaving this change up to you.